src/node_crypto.cc

```diff
@@ -627,7 +627,8 @@ void SecureContext::Init(const FunctionCallbackInfo<Value>& args) {
       max_version = TLS1_2_VERSION;
       method = TLS_client_method();
     } else {
-      THROW_ERR_TLS_INVALID_PROTOCOL_METHOD(env, "Unknown method");
+      const std::string msg("Unknown method: ");
+      THROW_ERR_TLS_INVALID_PROTOCOL_METHOD(env, (msg + * sslmethod).c_str());
```

Comment:
In case this looks like extraneous white space -- the cpp linter doesn't allow `*sslmethod` here, though it does in all the calls to `strcmp()` above.